test/units/testsuite-64.sh

```diff
@@ -346,11 +347,12 @@ EOF
             else
                 udevadm lock --device="${devices[$disk]}" sfdisk -q -X gpt "${devices[$disk]}" <"$partscript" &
             fi
+            running[$disk]=$!
         done
 
-        # Wait for the above sfdisk commands to be finished.
-        for disk in {0..9}; do
-            udevadm lock --device="${devices[$disk]}" true
+        for key in "${!running[@]}"; do
+            wait "${running[$key]}"
+            unset "running[$key]"
```

Comment:
I wonder if it wouldn't be easier to just call `wait` after the `for` loop above, since that should wait for all background processes (and it looks like that's what we're going for):

From `builtin(1)`:
> wait [-fn] [-p varname] [id ...]
>  ... If id is not given, wait waits for all running background jobs and the last-executed process substitution, if its process id is the same as $!, and  the  return  status  is zero.

Reply:
(Unless we're interested in exit codes of the respective `udeavdm lock` commands, in which case this is perfectly fine.)

Reply:
I think it is safer to check the exit status of `udevadm lock sfdisk`.